experimental/Schemes/RationalMap.jl

```diff
@@ -510,20 +510,29 @@ function pullback(phi::RationalMap, C::AbsAlgebraicCycle)
   R = coefficient_ring(C)
   comps = IdDict{IdealSheaf, elem_type(R)}()
   for I in components(C)
-    # Find a patch in Y on which this component is visible
-    all_V = [V for V in affine_charts(Y) if !isone(I(V))]
-    for V in all_V
-  # @show all_V
-  # min_var = minimum([ngens(OO(V)) for V in all_V])
-  # @show min_var
-  # all_V = [V for V in all_V if ngens(OO(V)) == min_var]
-  # @show all_V
-  # deg_bound = minimum([maximum([total_degree(lifted_numerator(g)) for g in gens(I(V))]) for V in all_V])
-  # @show deg_bound
-  # all_V = [V for V in all_V if minimum([total_degree(lifted_numerator(g)) for g in gens(I(V))]) == deg_bound]
-  # @show all_V
-  # V = first(all_V)
-  # @show findfirst(x->x===V, affine_charts(Y))
+    @vprint :RationalMap 1 "trying cheap pullback\n"
```

Comment:
```suggestion
    @vprintln :RationalMap 1 "trying cheap pullback"
```
This is even easier to read